db.js

```diff
@@ -1,16 +1,130 @@
 #!/usr/bin/env node
 /* eslint no-unused-expressions: off */
 
-function create() {
+// set up the connection with db
+const pg = require('pg');
+const Sequelize = require('sequelize');
+
+const db = new Sequelize('db_name', 'username', 'password', { // change database name, username, password
+  host: 'host', // change host
+  dialect: 'postgres',
+});
+
+
+
+// definition of the Tweets table
+const Tweet = db.define('tweets', {
+  id: {
+    type: Sequelize.STRING,
+    primaryKey: true
+  },
+  text: {
+    type: Sequelize.TEXT,
+  },
+  hashtag: {
+    type: Sequelize.ARRAY(Sequelize.STRING),
+  },
+  timestamp: {
+    type: Sequelize.DATE,
+  },
+  geo: {
+    type: Sequelize.STRING,
+  },
+  location: {
+    type: Sequelize.STRING,
+  },
+});
+
+// definition of the HashtagStats table
+const HashtagStats = db.define('hashtagstats', {
+  country: {
+    type: Sequelize.STRING,
+    primaryKey: true
+  },
+  hashtag: {
+    type: Sequelize.STRING,
+    primaryKey: true
+  },
+  count: {
+    type: Sequelize.INTEGER,
+    defaultValue: 1
+  },
+});
+
+// returns a list of all the trending hashtags and counties
```

Comment:
Typo ("counties"). Also, maybe use the Javadoc `/** */` convention I'm using in `util.js` for all the function-level comments.